lua/spd/server/sv_spd.lua

```diff
@@ -1,52 +1,61 @@
 local spd = spd or {}
 local coltbl = coltbl or {}
 
-function spdEntityRemoved(ent)
-	
+local function spdEntityRemoved(ent)
 	spd[ent:EntIndex()] = nil
 	coltbl[ent:EntIndex()] = nil
-	
 end
-
 hook.Add("EntityRemoved", "spdEntityRemovedHook", spdEntityRemoved)
 
-function spdEntityTakeDamage(ent, dmg)
+local function spdEntityTakeDamage(ent, dmg)
 	local entOwner = ent:CPPIGetOwner()
 
 	if not IsValid( entOwner ) then
 		return
 	end
-	
+
 	if GetConVar("spd_enabled"):GetInt() == 0 then
 		return
 	end
 
 	if ent:GetClass() ~= "prop_physics" then
 		return
 	end
-	
-	if GetConVar("spd_physicsdamage"):GetInt() == 0 and dmg:IsDamageType(DMG_CRUSH) then
-		return
+
+	if dmg:IsDamageType( DMG_CRUSH ) then
+	    local physicsDamage = GetConVar( "spd_physicsdamage" ):GetFloat()
+
+	    if physicsDamage == 0 then return end
```

Comment:
Is this check even necessary? Shouldn't the scale factor be 1 by default?

Reply:
This lets us disable the damage entirely and break out early

Reply:
Not having that there will accomplish the same thing though, won't it?

Reply:
It could, but this way we're 100% sure that any damage of this type isn't handled in any way by SPD